### PR TITLE
post-training example with fisheye dataset

### PR DESCRIPTION
**Pull request by SeungjunNah**
* post-training exampled added: fisheye dataset (AgiBotWorld-Alpha)
* gradient clipping added
* mock data updated to match real data video length
* mock dataloader num_workers set to 0 to avoid shared memory issues
* unused argument removed from `Predict2Video2WorldModelConfig`
* linting

### CODE REVIEW

**Review by pjannaty — APPROVED** [review]
LGTM, Seungjun!

File: cosmos_predict2/configs/base/experiment/agibot_head_center_fisheye_color.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                    rope_t_extrapolation_ratio=1.0,
+                    rope_w_extrapolation_ratio=2.0,
+                ),
+            ),
+        )
+    ),
+    model_parallel=dict(
+        context_parallel_size=2,
+    ),
+    dataloader_train=dataloader_train_agibot_head_center_fisheye_color,
```
[the remark this answers is not in the document]
Comment (SeungjunNah) [reply cosmos_predict2/configs/base/experiment/agibot_head_center_fisheye_color.py:102]: Let's get it done in a separate PR.

File: cosmos_predict2/configs/base/experiment/agibot_head_center_fisheye_color.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                guardrail_config=dict(enabled=False),
+                max_num_conditional_frames=1,
+                min_num_conditional_frames=1,
+            ),
+        )
+    ),
+    model_parallel=dict(
+        context_parallel_size=8,
+    ),
+    dataloader_train=dataloader_train_agibot_head_center_fisheye_color,
```
[the remark this answers is not in the document]
Comment (SeungjunNah) [reply cosmos_predict2/configs/base/experiment/agibot_head_center_fisheye_color.py:155]: Let's get it done in a separate PR.
